### PR TITLE
Add malva to dprint

### LINKED ISSUE #382: Add Malva to dprint hook
https://github.com/bakdata/kpops/pull/365#discussion_r1380359474

### PR DESCRIPTION
**Pull request by sujuka99**
closes #382

### CODE REVIEW

File: dprint.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "$schema": "https://dprint.dev/schemas/v0.json",
   "incremental": true,
   "markdown": {},
+  "malva": {},
   "includes": [
     "**/*.{md}"
```
Comment (disrupted) [inline dprint.json:7]: `css` (and possibly other) file extensions needs to be added here

**Review by disrupted — CHANGES_REQUESTED** [review]
please also run the new formatter

File: dprint.json [hunk 2]
```diff
@@ -3,9 +3,13 @@
...
   "markdown": {},
   "toml": {},
+  "malva": {},
   "includes": [
     "**/*.{md}",
-    "**/*.{toml}"
+    "**/*.{toml}",
+    "**/*.{css}",
+    "**/*.{scss}",
+    "**/*.{sass}"
```
Comment (raminqaf) [inline dprint.json:12]: are these necessary?

File: docs/docs/stylesheets/extra.css
Comment (raminqaf) [inline docs/docs/stylesheets/extra.css:1]: just double check the docs on the dev if everything is fine after this refactoring

File: dprint.json (hunk 2 above)
Comment (raminqaf) [reply dprint.json:12]: Then remove them please

Comment (sujuka99) [reply dprint.json]: removed

File: docs/docs/stylesheets/extra.css
Comment (sujuka99) [reply docs/docs/stylesheets/extra.css:1]: Looks good to me, I didn't check extensively, but from what I saw there aren't obvious problems

**Pull request merged**